packages/utilities/psammead-styles/src/font-families.js

```diff
@@ -2,41 +2,42 @@ const reithFallback = `Helvetica, Arial, sans-serif;`;
 const reithSans = `font-family: ReithSans, ${reithFallback}`;
 const reithSerif = `font-family: ReithSerif, ${reithFallback}`;
 
+const getFontStyleAndWeight = (style, weight) =>
```

Comment:
Nice one 👍🏻

Reply:
We could make it even simpler by having
```js
const styles = {
  bold: '/* css */',
  medium: '/* css */',
  normal: '/* css */'
}

// usage
export const news = {
sansRegular: `
  ${styles.normal}
`
}
```
Instead of having a function call to get the style and weight